From de490ecc83359f3a4a3ca46479b14ea732cbddaa Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Wed, 11 Jun 2014 11:53:05 -0400
Subject: [kernel] smp: fix generic_exec_single indentation

Message-id: <1402487594-26332-6-git-send-email-prarit@redhat.com>
Patchwork-id: 83633
O-Subject: [RHEL7.1 PATCH BZ 1105192 05/14] kernel: fix generic_exec_single indentation
Bugzilla: 1105192
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Neil Horman <nhorman@redhat.com>

commit ca5ecd64c2cdbcd316d789467147e732746f39fa
Author: Christoph Hellwig <hch@infradead.org>
Date:   Thu Nov 14 14:32:10 2013 -0800

    kernel: fix generic_exec_single indentation

    Signed-off-by: Christoph Hellwig <hch@lst.de>
    Cc: Jan Kara <jack@suse.cz>
    Cc: Jens Axboe <axboe@kernel.dk>
    Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
    Signed-off-by: Linus Torvalds <torvalds@linux-foundation.org>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/kernel/smp.c b/kernel/smp.c
index 069d20b..d9c5c49 100644
--- a/kernel/smp.c
+++ b/kernel/smp.c
@@ -140,8 +140,7 @@ static void csd_unlock(struct call_single_data *csd)
  * for execution on the given CPU. data must already have
  * ->func, ->info, and ->flags set.
  */
-static
-void generic_exec_single(int cpu, struct call_single_data *csd, int wait)
+static void generic_exec_single(int cpu, struct call_single_data *csd, int wait)
 {
  struct call_single_queue *dst = &per_cpu(call_single_queue, cpu);
  unsigned long flags;
-- 
1.7.1